admin/policy.md

```diff
@@ -15,7 +15,7 @@ One person speaks at a time. No person speaks for more than one minute when othe
  - It's important to thank members who are contributing to HSBNE as we wouldn't be where we are today without it. Take the time in a meeting to thank people for excellence at HSBNE.
 
 ## Site Inductions
-Before a new membership is accepted, the new member must complete a site induction as per our [general membership and payments policy](https://docs.google.com/document/d/1lgnfvc8fAFXcWZ4Sv9Pe9AE0ctgeGFhkOuM0QbNZqM8/edit). In addition to this requirement, in order to maintain membership, a member must have successfully completed a site induction within the last 12 months. To avoid any doubt, if a member has not completed a site induction within the last 12 months, their membership may be revoked until such time they complete a new site induction. Successfully completing a site induction for the purpose of this section, is defined in the general membership and payments policy.
+Before a new membership is accepted, the new member must complete a site induction as per our [general membership and payments policy](/membership). In addition to this requirement, in order to maintain membership, a member must have successfully completed a site induction within the last 12 months. To avoid any doubt, if a member has not completed a site induction within the last 12 months, their membership may be revoked until such time they complete a new site induction. Successfully completing a site induction for the purpose of this section, is defined in the general membership and payments policy.
```

Comment:
Revoking membership is a bit harsh, removing after-hours access and access to all RFID-enabled tools would be a more appropriate action for not completing a site induction within 12 months.